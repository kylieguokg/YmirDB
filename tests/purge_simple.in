purge a
set a 19 19 20 30
get a
purge a
get a
set a 20101
get a
purge a
get a
set wow 10
set oh 1 2 3
set dang 2305 39 1
list entries
list keys
purge wow
purge oh
purge dang
bye
