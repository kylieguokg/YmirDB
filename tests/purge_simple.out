> ok

> ok

> [19 19 20 30]

> ok

> no such key

> ok

> [20101]

> ok

> no such key

> ok

> ok

> ok

> dang [2305 39 1]
oh [1 2 3]
wow [10]

> dang
oh
wow

> ok

> ok

> ok

> bye
